fastlane/lib/fastlane/cli_tools_distributor.rb

```diff
@@ -45,7 +45,7 @@ def take_off
           print_bundle_exec_warning(is_slow: (Time.now - before_import_time > 3))
         end
 
-        unless (ENV['LANG'] || "").end_with?("UTF-8") || (ENV['LC_ALL'] || "").end_with?("UTF-8")
+        unless `locale chapmap` == "UTF-8"
```

Comment:
This does not work on macOS and I don't think this will work on Windows 😔 
```sh
➜  fastlane git:(pickfire-patch-1) locale chapmap
unknown keyword chapmap
```
which results in
```sh
➜  test-ios fl hey
[✔] 🚀
unknown keyword chapmap
[07:00:10]: WARNING: fastlane requires your locale to be set to UTF-8. To learn more go to https://docs.fastlane.tools/getting-started/ios/setup/#set-up-environment-variables
```

The "less elegant" but more platform agnostic approach would be to add separate check for `en_US.utf8` if we are trying to fix https://github.com/fastlane/fastlane/issues/13438

Reply:
why was it working on my macOS 🤔